cvat-ui/src/utils/platform.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+export function platform(): string {
+    let platformName = 'unknown';
+
+    if (window.isSecureContext && 'userAgentData' in navigator) {
+        // Use the new User-Agent Client Hints API when available in secure context
+        platformName = (navigator as any).userAgentData?.platform || navigator.platform;
```

Comment:
With just ``if .. else`` it may be rewritten with two returns. 'unknown' is not necessary in the beginning.
Additionally, I believe .toLowerCase() should not be in utility function. If outside code needs lowercase, better to do it outside.